exe/shell/main.go

```diff
@@ -278,15 +286,17 @@ func handleLine(line string) (out string, err error) {
 
 func cmdCmd() (out string, err error) {
 	out = pink(":help  ") + grey("Show available commands.\n")
-	out += pink(":address  ") + grey("Show host or active thread addresses.\n")
+	out += pink(":address  ") + grey("Show the host or active thread addresses.\n")
 	out += pink(":threads  ") + grey("Show threads.\n")
 	out += pink(":add <name>  ") + grey("Add a new thread with name.\n")
-	out += pink(":add <name> <address>  ") + grey("Add an existing thread with name at address.\n")
+	out += pink(":add <name> <address> <follow-key> <read-key>  ") + grey("Add an existing thread with name at address using follow and read keys.\n")
```

Comment:
Maybe mentioning in the description that keys should be base58 encoded

Reply:
👍